lib/LedgerSMB/PriceMatrix.pm

```diff
@@ -173,6 +124,11 @@ sub price_matrix {
                            - ($sellprice * ($mref->{pricebreak} / 100));
             }
             $ref->{sellprice} = $sellprice;
+            if ($mref->{qty} > $form->{qtycache}->{$ref->{id}}){
```

Comment:
It seems that this: https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/IS.pm#L1600
loop needs to be split in a loop loading from the database and a loop applying the prices from the price matrix.

I'm assuming there's something similar going on in the 'ir' and 'oe' sides of things.